// C-ERR

func main() {
    true()
}